react-frontend/public/index.html

```diff
@@ -25,9 +25,12 @@
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
     <title>Senzr</title>
-    <link rel="preconnect" href="https://fonts.gstatic.com">
-    <link href="https://fonts.googleapis.com/css2?family=Bebas+Neue&family=Montserrat:wght@300&display=swap"
-      rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Varela&display=swap"
```

Comment:
Great! Have you thought of what font weights that we need? Currently, it seems like we are loading all font weights which can cause a lot of performance issues on slower internet connections.

Suggestion is that we add:
- 300 (light)
- 400 (normal)
- 700 (bold)

If you look at the font you removed, you can see an example of how to choose specific font weight: `...&family=Montserrat:wght@300&...`

I think you will be able to do something like this: `https://fonts.googleapis.com/css2?family=Varela:Montserrat:wght@300;400;700&display=swap`

source: https://developers.google.com/fonts/docs/css2#individual_styles_such_as_weight

Reply:
Great input, thanks! I will look into this and let you review the changes.

Reply:
Added your suggested font-weight in the latest commit!